hyperopt/fmin.py

```diff
@@ -424,6 +424,16 @@ def fmin(fn, space, algo, max_evals, trials=None, rstate=None,
         if len(trials.trials) == 0:
             raise Exception("There are no evaluation tasks, cannot return argmin of task losses.")
         return trials.argmin
+        # Make sure the arguments are returned with the correct type
```

Comment:
I'm not sure this is doing anything if it goes after a return statement

Reply:
You're right. Thank you for spotting that.
Probably a bad fix of merge conflicts. I'll fix it as soon as I have a bit of time.

Reply:
Fixed.
Squashed both commits into a single commit.